ktpinblock/ktpinblock/build.gradle.kts

```diff
@@ -2,8 +2,12 @@ plugins {
     kotlin("multiplatform")
     kotlin("native.cocoapods")
     id("com.android.library")
+    id("maven-publish")
```

Comment:
We are not using Maven in our current web example, so it can be removed. It doesn't hurt to stay here though - see README.md for details.